### PR TITLE
implement integration with test rail as formatter

### PR DESCRIPTION
**Pull request by nomid0612**

### CODE REVIEW

File: src/testClientI.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import {testRailConfig} from "./testRailConfig";
+// @ts-ignore
+import { sendHttpRequest } from '@qavajs/steps-api';
```
Comment (AlexGalichenko) [inline src/testClientI.ts]: It will require installing @qavajs/steps-api. I would suggest to describe sendHttpRequest in this module

Comment (nomid0612) [reply src/testClientI.ts]: i thought about the same, but i see that we reuse some other modules, like memory

Comment (AlexGalichenko) [reply src/testClientI.ts]: as discussed in chat, lets eject sendHttpRequest into module?

File: src/testClientI.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        this.pass = params.pass;
+        const token: string = `Basic ${Buffer.from(`${(this.userName)}:${this.pass}`).toString('base64')}`;
+        const headers = {
+            'Content-Type': 'application/json;  charset=utf-8',
+            Authorization: token,
+        };
+        this.headers = headers;
+    }
+
+    async updateCaseStatusInRun(runId: number|string, caseId: number|string, requestBody: any) {
```
Comment (KirillBogdanets) [inline src/testClientI.ts]: please add returned type

**Comment by AlexGalichenko** [comment]
I also think that repo and package need to rename as testrail-formatter, to be consistent with other modules
@nomid0612 , @KirillBogdanets

File: src/testClientI.ts (hunk 1 above)
Comment (nomid0612) [reply src/testClientI.ts]: done

File: src/testClientI.ts (hunk 2 above)
Comment (nomid0612) [reply src/testClientI.ts]: added any return type

**Review by AlexGalichenko — APPROVED** [review]

**Pull request merged**